qiskit_experiments/test/mock_iq_helpers.py

```diff
@@ -181,9 +209,24 @@ def __init__(
         self.exp_helper_list = exp_helper_list
         self.exp_list = exp_list
 
-    def compute_probabilities(self, circuits: List[QuantumCircuit]) -> List[Dict[str, Any]]:
+    # pylint: disable=arguments-differ
```

Comment:
This suggests that something could be done better. Should the base class be changed? Should the variables `centers` and `widths` be internal variables set at `__init__`?